tests/test_parser.py

```diff
@@ -172,3 +172,101 @@ def test_include_invalid(self, include):
         """
         with pytest.raises(UnexpectedToken, match=r"Unexpected token"):
             parse_script(f"use {include};")
+
+    @pytest.mark.parametrize(
+        "stmt, expected_wires",
+        [
+            (
+                "gate MultiRot(x, y, z): RX(x) | [0]; RY(y) | [0]; RZ(z) | [0]; end;",
+                (0,),
+            ),
+            (
+                "gate MultiRot(x, y, z): RX(x) | [0]; RY(y) | [1]; RZ(z) | [2]; end;",
+                (0, 1, 2),
+            ),
```

Comment:
Can we also add a parametrization for
```suggestion
            (
                "gate MultiRot(x, y, z): RX(x) | [0]; RY(y) | [1]; RZ(z) | [2]; end;",
                (0, 1, 2),
            ),
            # Reverse order of the above.
            (
                "gate MultiRot(x, y, z): RX(x) | [2]; RY(y) | [1]; RZ(z) | [0]; end;",
                (0, 1, 2),
            ),
```